label_studio/io_storages/utils.py

```diff
@@ -109,3 +116,73 @@ def parse_range(range_header):
         end = ''
 
     return start, end
+
+
+@dataclass
+class StorageObjectParams:
+    task_data: dict
+    key: str
+    row_index: int | None = None
+    row_group: int | None = None
+
+    @classmethod
+    def bulk_create(
+        cls, task_datas: list[dict], key, row_idxs: list[int] | None = None, row_groups: list[int] | None = None
```

Comment:
I would prefer `row_indexes` - generally in favor of avoiding abbreviations in variable names unless they're truly ubiquitous.